playbooks/common/openshift-cluster/config.yml

```diff
@@ -42,6 +42,13 @@
     r_openshift_excluder_action: disable
     r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
+- name: Setup Docker
+  hosts: oo_all_hosts
```

Comment:
We won't have to configure docker on all hosts, such as on loadbalancer/haproxy or etcd hosts (when not containerized).